OpenEdXMobile/src/main/java/org/edx/mobile/module/db/IDatabase.java

```diff
@@ -478,4 +483,8 @@ Boolean isAnyVideoDownloadingInSubSection(DataCallback<Boolean> callback, String
      */
     boolean isUnitAccessed(DataCallback<Boolean> callback, String unitId);
 
+    /**
+     * Reset app directory
+     */
+    void resetDirectory();
```

Comment:
Please remove this method. This isn't linked with the Database instead move the actual implementation in the Storage and IStorage file.

Reply:
moved to FileUtiles